cosmovisor/cmd/cosmovisor/cmd/version.go

```diff
@@ -3,44 +3,46 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	"os"
 	"strings"
-	"time"
 
```

Comment:
Changes LGTM, but I'm weary of the package path: `cosmovisor/cmd/cosmovisor/cmd`.

Unless I'm mistaken, there really isn't a need for the latter `cmd` package, all the commands and the root handler can all exist under `cosmovisor/cmd/cosmovisor`. WDYT?

Reply:
I agree 👍🏾, changed!

The only thing is now everything is in the main package, which is not so great for testing, but here not problematic.